packages/openapi-generator/src/generator.ts

```diff
@@ -20,6 +27,10 @@ const logger = createLogger('openapi-generator');
  * @param options Options to configure generation.
  */
 export async function generate(options: GeneratorOptions): Promise<void> {
+  options.serviceMapping =
+    options.serviceMapping ||
```

Comment:
In the description you state:  By default it will be saved to/read from the input directory as "service-mapping.json".
So I would assume that the `options.serviceMapping` points to a file containing the service mapping and if this is not there it reads it from the file `service-mapping.json`. Compare also to the OData part and perhaps adjust the description if the current behavior the one also used in OData.